contracts/PGProject.sol

```diff
@@ -6,7 +6,7 @@ import "@openzeppelin/contracts/utils/Counters.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "hardhat/console.sol";
 
-contract Playground is ERC721Enumerable, Ownable {
+contract PGProject is ERC721Enumerable, Ownable {
```

Comment:
created for each project. users need to know the address of their project to access it on /deposit and /claim